app/javascript/src/lib/OWLanguageLinter.ts

```diff
@@ -295,6 +297,16 @@ function checkTranslations(content: string): void {
     const translateArguments = splitArgumentsString(content.substring(translateStartParenthesisIndex + 1, translateEndParenthesisIndex - 1))
     if (translateArguments.length === 0) continue
 
+    if (translateArguments.length > 1 && isStatic) {
+      diagnostics.push({
+        from: match.index + match[0].length + translateArguments[0].length,
```

Comment:
Could the diagnostic start from the second argument, instead of the comma after the first argument?
I can imagine that's a little harder to do since the argument splitting is handled by `splitArgumentsString`, but maybe you can think of some quick solution?

![diagnostic highlighting from the comma after the first argument, very ugly](https://github.com/user-attachments/assets/6231ba99-4133-45da-9ceb-cf5df893938e)